docs/guides/main.rst

```diff
@@ -93,24 +93,23 @@ Installing on Windows
 Installing beets on Windows can be tricky. Following these steps might help you
 get it right:
 
-1. If you don't have it, `install Python`_ (you want Python 3.6). On the last
-   screen, the installer gives you the option to "add Python to PATH." Check
-   this box. If you do that, you can skip the next step.
+1. If you don't have it, `install Python`_ (you want Python 3.6). During the 
+   installation, look for an option to "Add Python 3.6 to PATH". Check this box.
+   If you have done this, you can skip the next step.
 
-2. If you haven't done so already, set your ``PATH`` environment variable to
-   include Python and its scripts. To do so, you have to get the "Properties"
-   window for "My Computer", then choose the "Advanced" tab, then hit the
-   "Environment Variables" button, and then look for the ``PATH`` variable in
-   the table. Add the following to the end of the variable's value:
-   ``;C:\Python36;C:\Python36\Scripts``. You may need to adjust these paths to
-   point to your Python installation.
+2. Set your ``PATH`` environment variable to include Python and its scripts. 
+    To do so, you have to get the "Properties" window for "My Computer", then 
+    choose the "Advanced" tab, then hit the "Environment Variables" button, and 
+    then look for the ``PATH`` variable in the table. Add the following to the 
+    end of the variable's value: ``;C:\Python36;C:\Python36\Scripts``. You may 
+    need to adjust these paths to point to your Python installation.
```

Comment:
I'd like to keep the "if you haven't done so already," because I suspect some people will have already installed Python and already made this change.